site.yml

```diff
@@ -34,6 +34,9 @@
       when: serverspec.enabled|default("False")|bool
     - role: logging
       tags: ['common', 'logging']
+    - role: audit
+      tags: ['common', 'logging', 'audit']
+      when: audit.enabled|default('False')|bool
```

Comment:
Should we not be enabling the audit rules always, what we don't want to do is forward the syslog in the syslog forwarding is not enabled.